src/main/resources/templates/db_scripts/schema.json

```diff
@@ -185,6 +190,11 @@
       "tableName": "settings",
       "fromModuleVersion": "mod-organizations-storage-4.7.0",
       "withMetadata": true
+    },
+    {
+      "tableName": "banking_information",
```

Comment:
we should have organizationId as a foreign key and RMB will automatically create index for foreign key

Reply:
Added foreign key